### ISSUE TITLE
KT1 account currency cannot be transferred to the management account

### ISSUE THREAD

**Issue by mowenbin86** [Cryptonomic/Tezori#694]
Two days have passed, why can’t the coins baked in my KT1 account be transferred to the management account?

**Comment by mowenbin86** [comment]
Shows "Successfully started contract invocation",but no have operation ID,can not check on block explorer

**Comment by Pepesaiyajin** [comment]
Have the same issue too

**Comment by shaun19826** [comment]
@mowenbin86 hello mate, are you still having this issue ?

**Comment by shaun19826** [comment]
@Pepesaiyajin hello mate, are you still having this issue ?

**Comment by shaun19826** [comment]
@mowenbin86 Hello, Are you still having this issue or has your been resolved already ?

**Comment by shaun19826** [comment]
@Pepesaiyajin Hello, Are you still having this issue or has your been resolved already ?

**Comment by mowenbin86** [comment]
Yes, My main account does not have enough coins, the problem has been solved, thank you


| |
莫文滨
|
|
mowenbin86@163.com
|
签名由网易邮箱大师定制
On 7/28/2020 04:00，Pepesaiyajin<notifications@github.com> wrote：

Have the same issue too

**Comment by shaun19826** [comment]
Okay.
You can access your account on the galleon web wallet which has more
features than the galleon app, it also shows a list of bakers with high
payout options and low fees.

All you need to do is to import your fundraiser PDF doc (if you
participated) OR Restore your wallet using your private key (most preferred
and advisable) or 12 word backup phrase on the galleon web wallet on
www.galleon.network

You’ll have access to your funds upon a successful restoration.
